packages/manager/src/utilities/getAPIErrorFor.ts

```
@@ -4,6 +4,19 @@ export default (
   errorMap: { [index: string]: string },
   arr: APIError[] = []
 ) => (field: string): undefined | string => {
+  // One time during a sprint review I was showing off the new RDNS for routed
+  // ranges feature. When attempting to update the RDNS, the application
+  // crashed. Either something went wrong in then `.then()` handler (more
+  // likely), or we received an error response from the API we weren't prepared
+  // to handle (less likely). I was not able to reproduce this issue without
+  // manually throwing an error. Either way, this represents a chink in our
+  // armor, as we always assume `.catch()`s will be called with an API error
+  // response. This is an old error handling utility, but I wanted to add it
+  // as a safeguard so that this particular bug wouldn't happen again.
+  if (!Array.isArray(arr)) {
+    return;
```

This will swallow the error, which given that this is an old method may be the safest approach. Another thought would be to return `return field === 'none' ? 'An unknown error occurred.' : undefined;` so that getErrorFor('none') still sends back a generic error.

---

Good idea; just pushed up this change.